ci/requirements-dev-3.6.yml

```diff
@@ -26,7 +26,7 @@ dependencies:
   - requests
   - six
   - sqlalchemy>=1.0.0,<1.1.15
-  - thrift
+  - thrift>=0.10.0,<0.11.0
```

Comment:
is this orthogonal change?

Reply:
Yeah, tests were refusing to pass (and still are).

Reply:
Got rid of the pin.